### PR TITLE
Description of custom osc tool's table names

### PR DESCRIPTION
**Pull request by sunzhaoyang**
Add description of custom osc tool's temporary table name

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glorv** [comment]
LGTM

**Review by lance6716 — APPROVED** [review]

File: dm/dm-custom-online-ddl.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (ran-huang) [inline dm/dm-custom-online-ddl.md:1]: Please add this file to TOC.md

Comment (ran-huang) [reply dm/dm-custom-online-ddl.md:1]: Also, I think the content in this file will be better understood as a `## 其他 Online DDL 工具` section in `dm/feature-online-ddl.md`. If you merge this file with `dm/feature-online-ddl.md`, you don't have to repeat paragraphs like "一般会生成三种表" or "默认临时表命名规则".

**Review by ran-huang — APPROVED** [review]

**Pull request merged**